core/src/main/java/com/google/googlejavaformat/java/Main.java

```diff
@@ -207,6 +207,7 @@ private int formatFiles(ArgInfo argInfo, JavaFormatterOptions options) {
     int numThreads = Math.min(MAX_THREADS, argInfo.parameters.fileNamesFlag.size());
     ExecutorService executorService = Executors.newFixedThreadPool(numThreads);
 
+    Map<Path, String> inputs = new LinkedHashMap<>();
```

Comment:
I tend to keep the second map. It's a simple use-case, the caller has all information needed and the callable has no benefit from known if something changed.